qwesty.ino

```diff
@@ -287,11 +287,40 @@ void setOctave(uint8_t keyCode) {
 }
 
 void setRoot(uint8_t keyCode) {
-  for (uint8_t i = 0; i < 12; i++) {
-    if (keyCode == ROOT_KEYS[i]) {
-      state.root = i;
-      return;
+
+  if (keyCode == PS2_KEY_L_ARROW) {
+    if (state.root == 0) {
+      if (state.octave == -2) {
+      // We're at the lowest note in the lowest octave --- bail out. 
+      return; 
```

Comment:
Some goofy indentation, here. Not a big deal! TIP: If you're using Arduino IDE you can press Cmd+T (OR Ctrl+T I assume on PC/linux) to auto format the whole file